website/src/components/Layout.tsx

```diff
@@ -53,8 +53,8 @@ export const getDashboardLayout = (page: React.ReactElement) => (
       ]}
     >
       {page}
+      <Footer />
```

Comment:
This leaves the footer in a weird floating state as in the screenshot.  Can we get the footer to always be at the bottom?  Embedding it inside of the dashboard panel feels really weird.  Footers typically always live at the bottom below everything.

Reply:
The dashboard has two sections: a sticky sidebar and the content. If the footer was under both it would shrink the view of the content and sidebar which results in a less than ideal user experience. The reason it "floats" under the content is because it's positioned after the last object in the container. Once the container fills up with more widgets and the leaderboard is filled with the correct data it won't be floating anymore. It's just a consequence of the dashboard not being finished.

Think of how Chat GPT or instagram has their footer in the content container rather than under both their sidebar and content container. The footer doesn't necessarily have to be under every container and in a lot of cases isn't even shown in some parts of some apps.